### PR TITLE
Feature/plena reminder

### PR DESCRIPTION
**Pull request by LennyPenny**
notifies users interested in plena a private message one day before the plenum

### CODE REVIEW

**Comment by LennyPenny** [comment]
belasto :x

**Comment by LennyPenny** [comment]
bitte squashen :/

**Comment by LennyPenny** [comment]
review plz

File: src/tasks/plenum_reminder.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+def mark_plenum_announced(client: DiscourseStorageClient, plenum_date: datetime) -> None:
+    current_storage = client.storage.get(PLENUM_REMINDER_KEY)
+
+    current_storage[plenum_date] = "announced"
+
+    client.storage.put(PLENUM_REMINDER_KEY, current_storage)
+
+
+def is_day_before_plenum(date: datetime) -> bool:
```
Comment (jplatte) [inline src/tasks/plenum_reminder.py:47]: Always returns true? 🤨

File: src/tasks/plenum_reminder.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+def mark_plenum_announced(client: DiscourseStorageClient, plenum_date: datetime) -> None:
+    current_storage = client.storage.get(PLENUM_REMINDER_KEY)
+
+    current_storage[plenum_date] = 'announced'
+
+    client.storage.put(PLENUM_REMINDER_KEY, current_storage)
+
+
+def is_day_before_plenum(date: datetime) -> bool:
+    day_before_plenum = date-timedelta(1)
```
Comment (jplatte) [inline src/tasks/plenum_reminder.py:48]:
```python
return datetime.now().date() == (date - timedelta(1)).date()
```

Comment (jplatte) [reply src/tasks/plenum_reminder.py:48]: Current code only works if called at exactly `00:00:00`

**Pull request merged**